Tasks/UsePythonVersionV0/package.json

```diff
@@ -28,7 +28,9 @@
     "@types/mocha": "^5.2.7",
     "azure-pipelines-task-lib": "^3.1.0",
     "azure-pipelines-tasks-utility-common": "^3.0.3",
-    "azure-pipelines-tool-lib": "^1.0.2"
+    "azure-pipelines-tool-lib": "^1.1.1",
```

Comment:
Please update version to the relevant one at the moment

Reply:
Done, thanks